R/patient_util.R

```diff
@@ -47,15 +47,17 @@ Pt._FixDatatypes <- function(patients) {
   return(patients)
 }
 
+# Replaces blank names with "-"
 Pt._FixInvalidData <- function(patients) {
   patients$Nombre[patients$Nombre == ""] <- "-"
   patients$Apellido[patients$Apellido == ""] <- "-"
   return(patients)
 }
 
+# Expects that blank names have been replaced by "-"
 Pt._FilterUnsalvagableData <- function(patients) {
   patients <- patients[
-    !(patients$Nombre == "" & patients$Apellido == ""),
+    !(patients$Nombre == "-" & patients$Apellido == "-"),
```

Comment:
This was a bug -- there were a bunch of patients being created with "-" as both their first and last name.